Formula/freerdp.rb

```diff
@@ -55,6 +55,26 @@ def install
     system "cmake", "--install", "build"
   end
 
+  def caveats
+    # on_linux also applies Windows Subsystem for Linux (WSL)
+    on_linux do
+      <<~EOS
+        xfreerdp is an X11 application that requires an XServer be installed
+        and running. Lack of a running XServer will cause a "$DISPLAY" error.
+      EOS
+    end
+    on_macos do
+      <<~EOS
+        xfreerdp is an X11 application that requires an XServer be installed
+        and running. Lack of a running XServer will cause a "$DISPLAY" error.
+
+        XQuartz provides an XServer for macOS. The XQuartz can be installed
+        as a package from www.xquartz.org or as a Homebrew cask:
+          brew install --cask xquartz
+      EOS
+    end
```

Comment:
Does this work?
```suggestion
    extra = ""
    on_macos do
      extra = <<~EOS

        XQuartz provides an XServer for macOS. The XQuartz can be installed
        as a package from www.xquartz.org or as a Homebrew cask:
          brew install --cask xquartz
      EOS
    end

    <<~EOS
      xfreerdp is an X11 application that requires an XServer be installed
      and running. Lack of a running XServer will cause a "$DISPLAY" error.
      #{extra}
    EOS
```

Reply:
> Does this work?

yes.

Reply:
@SMillerDev Suggestion committed & passed tests. Anything else I need to do? Thanks.